app/utils/user/index.ts

```diff
@@ -380,3 +380,12 @@ export const getEmailIntervalTexts = (interval: string) => {
     };
     return intervalTexts[interval];
 };
+
+export const getLastPictureUpdate = (user: UserModel | UserProfile) => {
```

Comment:
The fact that the mobile app has both of these types is a bit scary to me, but I assume there's a good reason for it.

Could this function be made a bit clearer if we had a type guard to first determine if the object is a `UserModal` or `UserProfile` first before trying to figure out if it's a bot or not? I'm thinking something like this:
```ts
if (isUserProfile(user)) {
    return user.is_bot ? user.props.bot_last_icon_update : user.last_picture_update;
} else {
    return ...;
}
```

Reply:
Changed. Hopefully now it is a bit more clear :smile:

Reply:
Yep, that is. Thanks!